.github/workflows/inspektor-gadget.yml

```diff
@@ -222,7 +222,7 @@ jobs:
         path: /home/runner/work/inspektor-gadget/inspektor-gadget/${{ matrix.ig-target }}.tar.gz
 
   build-gadget-container-images:
-    name: Build gadget container images
+    name: gadget img
```

Comment:
As agreed for the new naming convention, it should be `ig-k8s img`. Same in scan and publish jobs.